packages/discovery-react-components/src/utils/document/documentUtils.ts

```diff
@@ -95,20 +106,32 @@ export function getTextNodeAndOffset(node: Node, strOffset: number): NodeOffset
     domOffset = Math.max(0, domOffset - adjustment);
   }
 
+  console.log(
```

Comment:
Remove this and the following console.log

Reply:
Oh, yeah, sorry, accidentally left in from debugging a test issue.